src/Data/String/CodePoints.purs

```diff
@@ -57,10 +58,27 @@ instance showCodePoint :: Show CodePoint where
 -- I would prefer that this smart constructor not need to exist and instead
 -- CodePoint just implements Enum, but the Enum module already depends on this
 -- one. To avoid the circular dependency, we just expose these two functions.
+-- |
+-- | ```purescript
+-- | >>> it = codePointFromInt 0x1D400 -- U+1D400 MATHEMATICAL BOLD CAPITAL A
+-- | Just (CodePoint 0x1D400) 
+-- | map singleton it
+-- |    == Just "𝐀"
+-- |
+-- | codePointFromInt 0x110000 -- does not correspond to a Unicode code point
+-- |    == Nothing
+-- | ```
+-- |
 codePointFromInt :: Int -> Maybe CodePoint
 codePointFromInt n | 0 <= n && n <= 0x10FFFF = Just (CodePoint n)
 codePointFromInt n = Nothing
 
+-- |
+-- | ```purescript
+-- | map codePointToInt $ codePointFromInt 0x1D400
```

Comment:
I can't remember if I've been applying this principle in the PRs of yours I already reviewed, but I think it might be nice to use parens instead of `$` in these examples, as if we use `$` it's just one more thing you have to know (admittedly a small thing, but I think it is probably worth it).

Reply:
previous PRs used `$`. I will probably go back and change that.